src/track/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 include_directories(
     ${Boost_INCLUDE_DIRS}
-    ${LIBUNWIND_INCLUDE_DIR}
```

Comment:
the libunwind handling should be squashed into the previous commit that introduces your separate unwinding backend